www/src/components/docs-table-of-contents.js

```diff
@@ -0,0 +1,61 @@
+import React from "react"
+import { Link } from "gatsby"
+import {
+  fontSizes,
+  colors,
+  zIndices,
+  shadows,
+  space,
+  mediaQueries,
+} from "../utils/presets"
+
+function createItems(items, location) {
+  return (
+    items &&
+    items.map(item => (
+      <li>
+        <Link to={location.pathname + item.url}>{item.title}</Link>
+        {item.items && (
+          <ul css={{ marginLeft: space[6] }}>
+            {createItems(item.items, location)}
+          </ul>
+        )}
+      </li>
+    ))
+  )
+}
+
+function TableOfContents({ page, location }) {
+  return page.tableOfContents.items ? (
+    <div
+      css={{
+        background: colors.white,
+        zIndex: zIndices.feedbackWidget,
+
+        [mediaQueries.xxl]: {
+          padding: space[6],
+          boxShadow: shadows.overlay,
+          float: `right`,
+          marginLeft: space[10],
+          position: `sticky`,
+          top: 130,
+        },
+      }}
+    >
+      <h1
```

Comment:
I think these should be `h2` tags since we already have an `h1` on the page for the title. Then it will also match the subheaders that it falls inline with on the smaller viewport size.

Reply:
Done 😄